style.css

```diff
@@ -0,0 +1,48 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+.app {    
+    background-color: #FFFFFF;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;       
+    max-width: 1440px;
+    max-height: 704px;     
+}
+
+.feature-details {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    max-width: 1110px;
+    max-height: 128px;
+    margin-left: 165px;
+    margin-top: 64px;    
```

Comment:
When aligning an item it should only be using flexbox and margin if necessary. The margin here looks like more on one side to make it cnetered. So change it accordingly.